.github/workflows/push.yml

```diff
@@ -0,0 +1,42 @@
+name: Build & Push Container Image
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      IMAGE: ${{ github.repository }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set image version latest
+        if: github.ref == 'refs/heads/master'
+        run: echo "VERSION=latest" >> ${GITHUB_ENV}
+      - name: Set image version from tag
+        if: startsWith(github.ref, 'refs/tags/v')
+        run: echo "VERSION=$(echo ${GITHUB_REF#refs/tags/})" >> ${GITHUB_ENV}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
```

Comment:
Why do we need QEMU and Buildx if we're only pushing 1 architecture (linux/amd64)?

Reply:
Good point. Copy :spaghetti: error